src/js/components/productSupplier/ProductSupplierListTable.jsx

```diff
@@ -116,33 +192,44 @@ const ProductSupplierListTable = ({ filterParams }) => {
   } = useProductSupplierListTableData(filterParams);
 
   return (
-    <ListTableWrapper>
-      <ListTableTitleWrapper>
-        <span>
-          <Translate id="react.productSupplier.listProductSources.label" defaultMessage="List Product Sources" />
-          &nbsp;
-          (
-          {tableData?.totalCount}
-          )
-        </span>
-      </ListTableTitleWrapper>
-      <DataTable
-        manual
-        sortable
-        ref={tableRef}
-        columns={columns}
-        data={tableData.data}
-        loading={loading}
-        defaultPageSize={10}
-        pages={tableData.pages}
-        totalData={tableData.totalCount}
-        onFetchData={onFetchHandler}
-        noDataText="No product sources match the given criteria"
-        footerComponent={() => (
-          <span className="title-text p-1 d-flex flex-1 justify-content-end" />
-        )}
+    <>
+      <CustomModal
+        titleLabel="react.productSupplier.deleteConfirmation.title.label"
+        defaultTitle="Are you sure?"
+        contentLabel="react.productSupplier.deleteConfirmation.content.label"
+        defaultContent="Are you sure you want to delete this Product Source?"
```

Comment:
maybe to make it shorter/more readable let's define the labels as an object that could look like:
```js
const labels = {
  title: {
   label: '...',
   defaultLabel: '...',
  },
  content: {
    label: '...',
    defaultLabel: '...',
  }
}